webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.jsx',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'public'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx$/,
+        include: path.join(__dirname, 'src'),
+        exclude: /(node_modules)/,
+        use: [
+          {
+            loader: 'babel-loader',
+            query: {
+              presets: ['es2015', 'react'],
```

Comment:
Same as the change in package.json, use the `env` preset here instead of `es2015` to save potential headaches.